pkgs/development/python-modules/orbax-checkpoint/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, flit-core
+, absl-py
+, etils
+, importlib-resources
+, jax
+, jaxlib
+, msgpack
+, nest-asyncio
+, numpy
+, protobuf
+, pyyaml
+, tensorstore
+, typing-extensions
+, flax
+, pytest
+, pytest-xdist
+, pytestCheckHook
+}:
+
+let self = buildPythonPackage rec {
+  pname = "orbax-checkpoint";
+  version = "0.4.2";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    pname = "orbax_checkpoint";
+    inherit version;
+    hash = "sha256-UaIfClZ76noFc5BUoCnMBtZ3PuRA3G5TCQHSI0KD0NI=";
+  };
+
+  nativeBuildInputs = [
+    flit-core
+  ];
+
+  propagatedBuildInputs = [
+    absl-py
+    etils
+    jax
+    jaxlib
+    importlib-resources
+    msgpack
+    nest-asyncio
+    numpy
+    protobuf
+    pyyaml
+    tensorstore
+    typing-extensions
+  ];
+
+  nativeCheckInputs = [
+    flax
+    pytest
+    pytest-xdist
+    pytestCheckHook
+  ];
+
+  doCheck = false;
+
+  passthru.tests = {
+    check = self.overridePythonAttrs (_: { doCheck = true; });
+  };
```

Comment:
Does it cause a circular import or something?
It would be good to have a short comment.

Reply:
Yes, circular import with `flax`. I added a comment !